max_corr: 3         #maxCorrespondences
max_iter: 100         #maxIterations
t_eps: 1e-8           #transformationEpsilon
fit_eps: 1e-2         #euclideanFitnessEpsilon
lidar_ang_covar: 7.78e-9  #These values loosely
lidar_lin_covar: 2.5e-4   #based on HDL32 resolution specs
covar_estimator: 0    #0 LUM, 1 Censi
res: 0.1              #voxel downsample filter, set to -1 not to use
multiscale_steps: 0   #How many times to match at a coarser scale
